lib/ansible/modules/net_tools/basics/uri.py

```diff
@@ -42,6 +42,13 @@
       - A password for the module to use for Digest, Basic or WSSE authentication.
     type: str
     aliases: [ password ]
+  decrypt:
+    description:
+      - Decrypts vaulted file with provided with appropriate password
+    type: bool
+    required: false
+    default: no
+    version_added: "2.10"
```

Comment:
To maintain consistency, keep just the `decrypt` header and `version_added: "2.10"` here and remove the rest.

Then add:

```
extends_documentation_fragment:
- decrypt
```

Reply:
Hi I have added the document as requested
```  
decrypt:
    description:
      - Decrypts vaulted file with provided with appropriate password
    extends_documentation_fragment:
      - decrypt
    version_added: "2.10"
```

Reply:
This isn't what I requested.

Reply:
@sivel  I'll work on this and update

Reply:
@sivel if I remove **default** from the documentation the build fails in shippable testing

Reply:
@sivel does this look good to you? You can let me know and we can change accordingly.